### PR TITLE
Fix rbac issue

### PR DESCRIPTION
**Pull request by richardwxn**
add namespace to work

### CODE REVIEW

File: deploy/service_account.yaml [hunk 1]
```diff
@@ -2,3 +2,4 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: istio-operator
+  namespace: istio-operator
```
Comment (sdake) [inline deploy/service_account.yaml]: this is redundant.  kubectl `-k` flag solves this with the kustomize.yaml code.

File: deploy/operator.yaml [hunk 2]
```diff
@@ -2,6 +2,7 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: istio-operator
+  namespace: istio-operator
```
Comment (sdake) [inline deploy/operator.yaml]: also redundant.

File: deploy/clusterrole.yaml [hunk 3]
```diff
@@ -59,6 +59,7 @@ rules:
   - configmaps
   - secrets
   - namespaces
+  - poddisruptionbudgets
```
Comment (sdake) [inline deploy/clusterrole.yaml]: this goes in `policy` not here.  This is already covered as well in the existing clusterrole.yaml:

```
- apiGroups:
  - policy
  resources:
  - poddisruptionbudgets
  verbs:
  - '*'
```

File: deploy/service_account.yaml (hunk 1 above)
Comment (ostromart) [reply deploy/service_account.yaml]: Let's keep it since we will have an operator init command soon.

Comment (sdake) [reply deploy/service_account.yaml]: I am not opposed to keeping the redundant and parts but I do want ppl to be able to deploy directly from the internet with just kubectl and no istioctl

Comment (sdake) [reply deploy/service_account.yaml]: I am not opposed to keeping the redundant and parts but I do want ppl to be able to deploy directly from the internet with just kubectl and no istioctl

File: deploy/clusterrole.yaml (hunk 3 above)
Comment (sdake) [reply deploy/clusterrole.yaml]: Please remove this from the or thx

Comment (sdake) [reply deploy/clusterrole.yaml]: Please remove this from the or thx

**Comment by sdake** [comment]
The namespaces are not redundant.  Please add the istio-operator namespace to all objects in the repo.  See PR https://github.com/istio/istio.io/pull/5142 for why this is necessary.

Cheers
-steve

File: deploy/clusterrole.yaml (hunk 3 above)
Comment (richardwxn) [reply deploy/clusterrole.yaml]: ok I didn't see it is added in policy group already. But somehow I am still seeing permission issue for pdb when testing.

Comment (sdake) [reply deploy/clusterrole.yaml]: can you paste the error?  I would like to see if in some way i fubared this section.. :)

**Comment by sdake** [comment]
@richardwxn service.yaml also in need of a namespace.

File: deploy/operator.yaml (hunk 2 above)
Comment (sdake) [reply deploy/operator.yaml]: ignore comment.

File: deploy/service_account.yaml (hunk 1 above)
Comment (sdake) [reply deploy/service_account.yaml]: ignore my comment.

**Comment by sdake** [comment]
subsumed by: https://github.com/istio/operator/pull/374

**Comment by sdake** [comment]
@richardwxn I can handle expanding RBAC for the operator here: https://github.com/istio/istio/issues/17841.  If you still suffer from a PDB issue, please paste the log message here or message to me on slack and I'll take a look.  If your satisfied with https://github.com/istio/operator/pull/374 - please close this issue and leave a comment.

Cheers
-steve

**Comment by richardwxn** [comment]
@sdake thanks for the clarification! I am closing this PR now as yours was merged already. I would send the error log to you on slack tmr.

**Pull request closed without merging**